src/test/java/qna/domain/AnswerRepositoryTest.java

```diff
@@ -0,0 +1,68 @@
+package qna.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+public class AnswerRepositoryTest {
+    @Autowired
+    AnswerRepository answerRepository;
```

Comment:
중요한 부분은 아니지만 접근 제한자를 붙여주시면 어떨까요?

```suggestion
    private AnswerRepository answerRepository;
```